src/styles/home_body.css

```diff
@@ -0,0 +1,54 @@
+@media only screen and (max-width: 720px) {
```

Comment:
You may combine these two media queries - the styles are the same for both `max-width: 720px` and `max-width: 1800px`